checkita-core/src/main/scala/org/checkita/dqf/storage/Managers.scala

```diff
@@ -270,19 +271,24 @@ object Managers {
     
     def saveResults[R <: DQEntity : TypeTag](results: Seq[R])(implicit ops: tables.DQTableOps[R]): String = {
       val (targetTable, _, schema) = getBasicVars[R](ops)
-      val curJobId = results.head.jobId
+      results match {
```

Comment:
Just some code style comment. I think pattern match is excessive here. Lets replace with:

```scala
if (results.isEmpty) "Nothing to save" else { ... }
```

We have the same block in `DqJdbcStorageManager`

Reply:
Done